istio/tests/Makefile

```diff
@@ -0,0 +1,17 @@
+KUBE_VERSION=v1.9.4
+NGINMESH_VERSION=0.6.0
+WRK_VERSION=3.1.0
+GIT_COMMIT=$(shell git rev-parse --short HEAD)
+TEST_IMAGE_NAME=bookinfo_testing
+REPO=gcr.io/$(GCLOUD_PROJECT)
+release: REPO=nginmesh
+TAG=$(NGINMESH_VERSION)-$(GIT_COMMIT)
+TAG_LATEST=$(NGINMESH_VERSION)
+
+release:
```

Comment:
don't duplicate the make rule.  have a single rule but override using different target. please see agent makefile as example